wagtailspace/settings/base.py

```diff
@@ -130,3 +130,8 @@
 }
 
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
+
+WAGTAILSPACE_REGISTRATION_NOTIFICATIONS = [
```

Comment:
Why not set this based on an environment variable, then we can configure it in Heroku?

Reply:
Good call. This should do the trick, and return an empty list if the environment variable isn't populated, guaranteeing the setting will always be a list.

Reply:
That works for me!